src/apps/Registration.js

```diff
@@ -56,7 +57,15 @@ const RegistrationComponent = (props) => {
                         .auth()
                         .createUserWithEmailAndPassword(email, pass)
                         .then((user) => {
```

Comment:
This was my mistake. Let's change it to 
```suggestion
                        .then(({user}) => {
```